sysbuild/CMakeLists.txt

```diff
@@ -94,6 +94,10 @@ function(include_fw_zip)
   include(${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/zip.cmake)
 endfunction()
 
+function(include_b0_packaging)
```

Comment:
No actually this was done because you submitted the original for `include_provision_hex` which did it this way (and that file actually has its own function that gets called inside of that file)

Reply:
and `include_provision_hex()` was exactly done like that because of scoping, because the `provision_hex.cmake` file imports the kconfig and thus defines several `CONFIG_` settings which would be polluting the scope.
https://github.com/nrfconnect/sdk-nrf/blob/86373aabea1531be83d01e6d0c122f047959dc23/subsys/bootloader/cmake/provision_hex.cmake#L16

But agreed, the `include_provision_hex()` should actually have a comment in this regard.

Generally there are no reason for such a pattern, but in the case of the `provision_hex.cmake`, then that file is shared between sysbuild and parent/child builds.